nuxt.config.js

```diff
@@ -88,7 +88,7 @@ export default {
       process.env.SEQUENCESCAPE_BASE_URL || 'http://sequencescape'
   },
   publicRuntimeConfig: {
-    asynchronous: true,
+    asynchronous: process.env.ASYNCHRONOUS,
```

Comment:
will need a corresponding change in the deployment project for this?

Reply:
Yes but it will allow you to set asynchronous to false local testing. I thought it was an improvement?

Reply:
Sure I was just flagging it so we remember.